src/player.cpp

```diff
@@ -3073,15 +3058,32 @@ Strength - 4;    Dexterity - 4;    Intelligence - 4;    Perception - 4"));
             fold_and_print( w_info, 0, 1, FULL_SCREEN_WIDTH - 2, c_magenta, s );
             wrefresh(w_info);
 
+            // which of the four paired body parts should get combined?
+            // TODO deduplicate this with print_encumbrance()
+            bool combined_limbs[4] = {}; // all false
+            for (size_t i = 0+4; i<12-1; i+=2) {
```

Comment:
This might be caused by the existing code, but what's with all the magic values? 12, 4, 10, 2*6, etc are repeated throughout the diff. Do they always represent the same thing? They should probably be converted to changed constants.

This would also make it much easier to follow the logic. For example here: why iterator over 4,6,8,10? What are those magic values? Apparently they are body part ids? Maybe this should iterate over an array of those ids instead.